signwriting_transcription/pose_to_signwriting/data/datasets_pose.py

```diff
@@ -12,32 +12,38 @@ def fsw_cut(fsw_text: str) -> str:
     return fsw_text
 
 
-def ms2frame(ms, frame_rate) -> int:
-    ms = int(ms)
-    return int(ms / 1000 * frame_rate)
+def ms2frame(mil_sec, frame_rate) -> int:
```

Comment:
why `mil_sec`, is `ms` not clear enough? if not, use `milliseconds` since it is one word.
if the linter screams at you - add `good-names = ["ms"]` to the pylint settings